### PR TITLE
Update Enable-CsOnlineDialInConferencingUser.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/office-docs-powershell/issues/2933

### CODE REVIEW

File: skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ If the bridge information isn't provided and the tenant uses multiple audio conf
 
 The audio conferencing provider name and domain information is automatically set for the user when they are enabled for audio conferencing.
 
+> [!NOTE]
+> If your conferencing provider is Microsoft, your users' conference IDs are set to Dynamic Only by default. Unfortunately, there's no ability to change it in the Skype for Business Admin Center or using Windows Powershell to become static, as this is now unsupported. Conference IDs are automatically set only for Skype for Business users enabled for Audio Conferencing.
```
[the remark this answers is not in the document]
Comment (dariomws306) [reply skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md]: Ok, I have copied and pasted from another article:
https://docs.microsoft.com/skypeforbusiness/audio-conferencing-in-office-365/reset-a-conference-id-for-a-user
But I think both ways are fine.

[the remark this answers is not in the document]
Comment (kenwith) [reply skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md]: Can we update that other article too?

[the remark this answers is not in the document]
Comment (dariomws306) [reply skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md]: Yes sure, I can do that.

[the remark this answers is not in the document]
Comment (kenwith) [reply skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md]: Ok, try the Commit Suggestion above. I think the "Resolve Conversation" is if you decide not to make the change.. and when we done talking you click that and it 'resolves' this whether you commit suggested change or not.

**Comment by dariomws306** [comment]
Done, I have applied to the other article too:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/1739

**Pull request merged**